### ISSUE TITLE
"Quick Tags" doesn't show a live list of tags while typing

### ISSUE THREAD

**Issue by Jectech** [standardnotes/forum#1303]
**Describe the bug**
If I enable the "Quick Tags" extension and start entering a tag, no suggestion will pop up. When pressing enter on my keyboard, nothing happen and no tags are created. I tried with the Desktop client and web, same result.  When I uninstall the “Quick Tags extension”, I can create tags and nested tags, and the note will appear.

**To Reproduce**
1. Ensure "quick tags" and “folders” extension are activated
2. Make a new note
3. Start adding a tag to the tag field until the text overflows

**Expected behavior**
I expect that a live list of tags will show up when typing, and be able to select one, or create a new one.

**Screenshots**
When I try to add a tag:
![image](https://user-images.githubusercontent.com/83036878/115897752-0bb11300-a45d-11eb-9bc5-9277abff1943.png)

When I uninstall the "Quick Tags" extension:
![image](https://user-images.githubusercontent.com/83036878/115897821-21bed380-a45d-11eb-840a-ad75492ac58f.png)
![image](https://user-images.githubusercontent.com/83036878/115897833-25525a80-a45d-11eb-85eb-b5f5de516e63.png)

What's the expected behavior:
![image](https://user-images.githubusercontent.com/83036878/115898006-5b8fda00-a45d-11eb-8bae-962395151f53.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 v1909
 - Browser: chrome 90.0.4430.85
 - Standard note Version: 3.6.7

**Additional context**
I'm using bold editor, and the quick tags and folders extensions. I got the same problem on another computer, running windows 10 with firefox 88.0. I tried also to clear the cache and reload. I also tried to reinstall the app on my laptop.

**Comment by JaspalSuri** [comment]
Hi @Jectech, sorry to hear that.

I'm afraid that I haven't been able to reproduce the issue. Can you try the following and let us know if it helps?

### Refresh the Extensions repo
1. Uninstall the **Quick Tags** and **Folders** extensions
2. Click on the **Refresh** button at the top of the **Extensions** menu
3. Reinstall the **Quick Tags** and **Folders** extensions

### Reset the app
1. Uninstall the **Quick Tags** and **Folders** extensions
2. Sign out, click on the **Account** menu again, and click **Clear Session Data**. Do the same for the web app, just in case.
3. Use the steps in this [support article](https://docs.standardnotes.org/troubleshooting/reset-apps/) for resetting the desktop app
4. Close the application
5. Restart the application
6. Sign in
7. Reinstall the **Quick Tags** and **Folders** extensions

If neither work, please open the developer tools (via Menu button > View > Toggle Developer Tools), click on the **Console** tab, create a couple of tags on a new note, and then send us a screenshot of any errors that appear. The errors will be highlighted in red and you may need to expand them to reveal more info.

Please let us know how that goes!

**Comment by Jectech** [comment]
Hi @JaspalSuri ! Thank you for your reply and support with this issue.

I tried to refresh the Extension repo, and to Reset the app, but it hasn't fix the issue. I still canno't get a list of tags while typing, or add a new tag

Here is a screenshot of the error showing up in the developer tool, when I tried to add a tag to a new note (I have an error for every character that I type):
![Standardnote Quick Tags error screenshot1](https://user-images.githubusercontent.com/83036878/116007320-bca2e380-a5fe-11eb-84ce-50b96ed14682.png)

When I create a nest tag in a folder, I get those 2 errors:
![Standardnote Quick Tags error screenshot2](https://user-images.githubusercontent.com/83036878/116007460-5e2a3500-a5ff-11eb-968b-caa04204a379.png)

Also, I'm not sure if this is link to the issue, but when I started the app, I got the following message popping up:
![Standardnote Quick Tags error screenshot3](https://user-images.githubusercontent.com/83036878/116007553-ce38bb00-a5ff-11eb-9809-2583bb990371.png)

**Comment by JaspalSuri** [comment]
Hi @Jectech,

I'm so sorry to hear that neither suggestions worked. Thank you for those screenshots! 

One last thing that comes to mind that might be worth trying is to uninstall both Extensions, [force reset your Extensions repo](https://standardnotes.org/help/41/my-extensions-appear-as-expired-even-though-my-subscription-is-still-valid) (see **Option 2**), reinsert and reapply your Extended code, and then reinstall them.

Please let us know how that goes!

**Comment by Jectech** [comment]
Hi @JaspalSuri,

Thanks for your reply. I just tried the force reset approach and I got the same error. I tried twice, the first time by uninstalling all the extensions, the second time by uninstalling only those 2. I also updated the app to the latest version (3.6.8) after that.

This is the screenshot I got from the error in the developer tool. Same as the one above from what I see.
![Standardnote Quick Tags error screenshot4](https://user-images.githubusercontent.com/83036878/116302794-2652f700-a791-11eb-8a4d-86019394d927.png)
